includes/functions.php

```diff
@@ -530,14 +530,36 @@ function utime()
   return $sec + $usec;
 }
 
+function getpollergroup($poller_group='0')
+{
+  //Is poller group an integer
+  if (is_int($poller_group) || ctype_digit($poller_group)) {
+    return $poller_group;
+  } else {
+    //Check if it contains a comma
+    if (strpos($poller_group,',')!== FALSE) {
+      //If it has a comma use the first element as the poller group
+      $poller_group=explode(',',$poller_group)[0];
+      return getpollergroup($poller_group);
```

Comment:
This is just a more expensive way of doing `return $poller_group`. Or am I missing something?